projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -83,6 +85,9 @@ public function get_item_permissions_check( $request ) { // phpcs:ignore Generic
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_item( $request ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter, VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		if ( ! ( new Host() )->is_wpcom_platform() && get_option( 'wpcom_admin_interface' ) !== 'wp-admin' ) {
```

Comment:
My understanding of the comment [here](https://github.com/Automattic/jetpack/pull/38804/files#r1745498954) is that we don't need to check for the `wpcom_admin_interface` option. Removing it in testing makes no difference to the admin menu, which works properly again with this PR.

```suggestion
		if ( ! ( new Host() )->is_wpcom_platform() ) {
```

Reply:
I was two minutes too slow in my review :)